.eslintrc.json

```diff
@@ -2,5 +2,8 @@
   "extends": "brightspace/node-config",
   "parserOptions": {
     "sourceType": "module"
+  },
+  "env": {
+    "mocha": true
```

Comment:
Feels like this should live in an `.eslintrc.json` in `test`, but I assume this is what @dlockhart said he'd play with in his PR - not sure where that comment went

Reply:
Yeah I'll sort it out in mine once this merges.